### ISSUE TITLE
error

### ISSUE THREAD

**Issue by lkdnvc** [avocode/avocode#1484]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.14.4`
Avocode UID: 153762
```
TypeError: Cannot read property 'get' of null
    at e.value (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1545775)
    at e.value (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1545693)
    at t._handleActivity (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1797920)
    at emitOne (events.js:101:20)
    at t.emit (events.js:188:7)
    at SockJS.n._handleSocketMessage [as onmessage] (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1313643)
    at SockJS.EventTarget.dispatchEvent (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/sockjs-client/lib/event/eventtarget.js:51:20)
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/sockjs-client/lib/main.js:274:16
    at Array.forEach (native)
    at SockJS._transportMessage (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/sockjs-client/lib/main.js:272:17)
    at emitOne (events.js:96:13)
    at WebSocketTransport.emit (events.js:188:7)
    at Client.WebSocketTransport.ws.onmessage (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/sockjs-client/lib/transport/websocket.js:35:10)
    at Client.dispatchEvent (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/faye-websocket/lib/faye/websocket/api/event_target.js:22:30)
    at Client._receiveMessage (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/faye-websocket/lib/faye/websocket/api.js:150:10)
    at Client.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/faye-websocket/lib/faye/websocket/api.js:34:49)
    at emitOne (events.js:101:20)
    at Client.emit (events.js:188:7)
    at Client.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/websocket-driver/lib/websocket/driver/hybi.js:451:14)
    at pipe (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/websocket-extensions/lib/pipeline/index.js:37:40)
    at Pipeline._loop (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/websocket-extensions/lib/pipeline/index.js:44:3)
    at Pipeline.processIncomingMessage (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/websocket-extensions/lib/pipeline/index.js:13:8)
    at Extensions.processIncomingMessage (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/websocket-extensions/lib/websocket_extensions.js:133:20)
    at Client._emitMessage (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/websocket-driver/lib/websocket/driver/hybi.js:442:22)
    at Client._emitFrame (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/websocket-driver/lib/websocket/driver/hybi.js:405:19)
    at Client.parse (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/websocket-driver/lib/websocket/driver/hybi.js:141:18)
    at Client.parse (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/websocket-driver/lib/websocket/driver/client.js:62:58)
    at IO.write (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/websocket-driver/lib/websocket/streams.js:80:16)
    at TLSSocket.ondata (_stream_readable.js:555:20)
    at emitOne (events.js:96:13)
    at TLSSocket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at TLSSocket.Readable.push (_stream_readable.js:134:10)
    at TLSWrap.onread (net.js:551:20)
```

**Comment by petrbrzek** [comment]
@lkdnvc thanks for reporting the issue. We fixed this bug in our dev branch.

**Comment by jakubzitny** [comment]
We've just released `2.14.5` with a fix for this bug, please update the app and let us know if you have any more problems 😉
